routers/name_router.py

```
@@ -0,0 +1,43 @@
+def get_names(handler: NameHandler = Depends(get_name_handler)):
+    return handler.get_names()
+
+
+@name_router.get("/{name}")
+def get_name_by_name(name: str, handler: NameHandler = Depends(get_name_handler)):
+    return handler.get_name_by_name(name)
+
+
+@name_router.put("/{name}")
+def change_name(
+    used_id: str, new_name: str, handler: NameHandler = Depends(get_name_handler)
```

Ruff (0.8.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)